examples/AspNet/Examples.AspNet.csproj

```diff
@@ -79,6 +79,7 @@
     <PackageReference Include="Microsoft.AspNet.WebApi.WebHost" Version="$(MicrosoftAspNetWebApiWebHostPkgVer)" />
     <PackageReference Include="Microsoft.AspNet.Mvc" Version="$(MicrosoftAspNetMvcPkgVer)" />
     <PackageReference Include="Microsoft.AspNet.WebPages" Version="$(MicrosoftAspNetWebPagesPkgVer)" />
+    <PackageReference Include="Microsoft.VisualBasic" Version="10.3.0" />
```

Comment:
Is this intentional?

Reply:
On my box I couldn't build without this. It seems like a missing reference. OR are we saying we need build box to have prior setup with VisualBasic in the global cache?

Reply:
lets remove it from this PR, and open a sep. issue since you faced this. (never seen anyone reporintg this so far, so I guess it might be something in your machine as well.)